packages/examples/packages/jsx/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metamask/jsx-example-snap",
-  "version": "1.1.0",
+  "version": "1.1.1",
```

Comment:
Please revert the version update here, we'll update it once we release!

Reply:
Done!